hq/conf/application.conf

```diff
@@ -20,8 +20,9 @@
 # ~~~~~
 # The secret key is used to sign Play's session cookie.
 # This must be changed for production, but we don't recommend you change it in this file.
-play.crypto.secret = ${APPLICATION_SECRET}
-application.secret = ${APPLICATION_SECRET}
+play.http.secret.key ="skkqY6yoYpT:tk_lS`=/]El/KhI[VlE<;FPRrp^v[j_b^]rbEl@68`/E<^MT@xwo"
```

Comment:
? Is this supposed to be here?

Reply:
i didn't realise we had a `APPLICATION_SECRET` key in the local `~/.gu/security-hq.local.conf`. Removing the default one.